src/views/InvolvementProfile/components/Membership/components/AdminCard/components/RequestsReceived/index.jsx

```diff
@@ -7,28 +7,13 @@ import {
   ListItemText,
   Typography,
 } from '@mui/material';
-import makeStyles from '@mui/styles/makeStyles';
 import { formatDistanceToNow } from 'date-fns';
 import { Fragment, useEffect, useState } from 'react';
```

Comment:
```suggestion
import { Fragment, useEffect, useState } from 'react';
```

^`Fragment` import is unnecessary

```
<>
...
</>
``` 
^ this is a react fragment.

Reply:
This is needed for `key={request.RequestID}`